### PR TITLE
Remove unneeded checks, fix for Matrix, and new tests!

### PR DESCRIPTION
**Pull request by glennguy**

### CODE REVIEW

File: lib/drmhelper/tests/fakes.py [hunk 1]
```diff
@@ -152,3 +152,17 @@ def getAddonInfo(self, key):
...
+TRANS_PATH_ARGS = [
+    "addon.getSetting('DECRYPTERPATH')",
+    'special://xbmcbinaddons/inputstream.adaptive',
+    'special://home/'
+]
+
+TRANSLATED_PATHS = {
+    'Linux': ['/storage/.kodi/cdm',
+              '/storage/.kodi/addons/inputstream.adaptive'],
+    'Windows': ['c:/users/user/appdata/roaming/Kodi/cdm',
```
Comment (andybotting) [inline lib/drmhelper/tests/fakes.py]: Want to make this capital C:/ instead?

Interesting the slashes are that way too

File: lib/drmhelper/tests/test_helper.py [hunk 2]
```diff
@@ -130,55 +139,84 @@ def test_get_wvcdm_filename(self):
...
-        with mock.patch.object(helper.DRMHelper, '_get_latest_ia_version',
-                               return_value={'ver': '0.0.1'}):
-            fake_addon = fakes.FakeAddon()
-            h = helper.DRMHelper()
-            result = h._is_ia_current(fake_addon, latest=True)
-            self.assertTrue(result)
+    @responses.activate
+    def test_get_wvcdm_current_ver(self):
+        responses.add(responses.GET, config.CMD_CURRENT_VERSION_URL,
+                      body='1.4.9.1088')
```
Comment (andybotting) [inline lib/drmhelper/tests/test_helper.py]: I'd actually be tempted here to use some dummy version number, so it's obvious that this is only for testing, like 9.9.9.9999

**Review by andybotting — COMMENTED** [review]
Really nice. Love your tests too.

File: lib/drmhelper/tests/fakes.py (hunk 1 above)
Comment (glennguy) [reply lib/drmhelper/tests/fakes.py]: Yeah win/python is happy with forward slashes that way you don't accidentally escape... `'c:\users\nelly"` is a disaster whereas `'c:\users\helly"` is perfectly fine. Double slash works but forward is nicer I think.

I'll make capital C but Windows love it/hate it doesn't care for case when asking for a path or file. Backwards compatibility with DOS I guess.

**Pull request merged**